<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DS-38607723D16D" name="test.java.CompleteLibraryTest" tests="10" failures="0" timestamp="4 Aug 2015 10:38:10 GMT" time="29.138" errors="1">
  <testcase name="testSearchMyNotes" time="2.956" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchByeText" time="2.926" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchMyNotes" time="2.493" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchByContents" time="3.147" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchByeText" time="1.692" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchByContents" time="6.434" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testVerifyUIOfCompleteLibraryPage" time="0.381" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchByeText" time="2.246" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchByContents" time="4.985" classname="test.java.CompleteLibraryTest"/>
  <testcase name="testSearchMyNotes" time="1.878" classname="test.java.CompleteLibraryTest">
    <error type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at test.java.CompleteLibraryTest.testSearchMyNotes(CompleteLibraryTest.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- testSearchMyNotes -->
</testsuite> <!-- test.java.CompleteLibraryTest -->
